supporting_scripts/extract_number_of_server_starts.sh

```diff
@@ -9,8 +9,8 @@ then
   exit 1
 fi
 
-if [[ $numberOfStarts -gt 5 ]]
+if [[ $numberOfStarts -gt 4 ]]
```

Comment:
```suggestion
if [[ $numberOfStarts -ne 4 ]]
```
Checking for non-equal (with an adapted error message) would also alert us in the case that was undetected previously.